README.md

```diff
@@ -64,6 +64,18 @@ The follow options can be provided to this GitHub Action.
       Example: <code>formatters codeqa</code>.
     </td>
   </tr>
+  <tr>
+    <td>
+      <code>extra-args</code>
+      <br>
+      (default:&nbsp;<code>""</code>)
+    </td>
+    <td>
+      Extra arguments to pass to all sessions. If left empty, no additional arguments will be passed
```

Comment:
This should be the exact text in https://github.com/ansible-community/github-action-run-nox/blob/main/action.yml#L20-L21:
```suggestion
      Extra arguments to be passed to nox sessions. If empty, no extra arguments are passed. Can be a space-separated list of arguments.
```